core/storage/user/gae_models_test.py

```diff
@@ -443,3 +446,52 @@ def test_get_categories_where_user_can_review(self):
         self.assertIn('category1', score_categories)
         self.assertIn('category3', score_categories)
         self.assertNotIn('category2', score_categories)
+
+
+class UserSubscriptionsModelTests(test_utils.GenericTestBase):
+    """Tests for UserSubscriptionsModel."""
+    USER_ID_1 = 'user_id_1'
+    USER_ID_2 = 'user_id_2'
+    CREATOR_IDS = ['4', '8', '16']
+    COLLECTION_IDS = ['23', '42', '4']
+    ACTIVITY_IDS = ['8', '16', '23']
+    GENERAL_FEEDBACK_THREAD_IDS = ['42', '4', '8']
+
+    def setUp(self):
+        """Set up user models in datastore for use in testing."""
+        super(UserSubscriptionsModelTests, self).setUp()
+        user_models.UserSubscriptionsModel(id=self.USER_ID_1).put()
+
+        user_model_2 = user_models.UserSubscriptionsModel(id=self.USER_ID_2)
```

Comment:
If you can, it would be preferable to create these by more natural methods, i.e. through functions in the domain layer.

Reply:
@ulloaluis

Reply:
@seanlip PTAL
Can you elaborate on "through functions in the domain layer," as I don't necessarily see how the way I construct my model is different from how Varun constructs his.
Also, can I ask why this is necessary? Regardless of whether it's constructed naturally or not, export_data will create a dictionary with only the specific fields I reference in export_data(user_id). This example feels distinct from my other PR, where I had a test where I was checking for formatting (which I don't do here).

Reply:
Per email I sent, I'm reverting this comment (we can discuss more in our meeting). Sorry about that!

Reply:
@seanlip @ulloaluis marking this as resolved for clean up purposes. If there is still some action either of you would like to take feel free to reopen